test/geoparquet/disabled.test

```diff
@@ -0,0 +1,99 @@
+# name: test/geoparquet/disabled.test
+# group: [geoparquet]
+
+require spatial
+
+require parquet
```

Comment:
Can we add tests when spatial is not loaded as well?

Reply:
Done!